assets/dev/js/frontend/frontend.js

```diff
@@ -7,7 +7,7 @@ import YouTubeApiLoader from './utils/video-api/youtube-loader';
 import VimeoApiLoader from './utils/video-api/vimeo-loader';
 import BaseVideoLoader from './utils/video-api/base-loader';
 import URLActions from './utils/url-actions';
-import Swiper from './utils/swiper';
+import SwiperHandler from './utils/swiper';
```

Comment:
We need to be able to distinguish between the Swiper library class name and our internal handler class name.